minecraft_servers/royalpixels/manifest.json

```diff
@@ -0,0 +1,45 @@
+{
+  "server_name": "royalpixels",
+  "nice_name": "Royalpixels",
+  "direct_ip": "mc.royalpixels.de",
+
+  "supported_languages": [
+     "en",
+     "de"
+  ],
+  "social": {
+    "web": "https://royalpixels.de",
+    "web_shop": "https://shop.royalpixels.de",
+    "twitter": "RoyalPixelsDE",
+    "tiktok": "royalpixels",
+    "instagram": "royalpixels.de",
+    "discord": "https://discord.gg/royalpixels",
+    "youtube": "https://www.youtube.com/RoyalPixels",
+    "teamspeak": "ts.royalpixels.de"
+  },
+  "gamemodes": {
+    "skywarsffa": {
+      "name": "SkyWarsFFA", 
+      "color": "#FFA500"
+    },
+    "eventsystem": {
+      "name": "EventSystem",
+      "color": "#ADD8E6"
+    }
+  },
+  "discord": {
+    "server_id": 344516118356033536,
+    "rename_to_minecraft_name": true
```

Comment:
Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.